spaceship/lib/spaceship/tunes/build.rb

```diff
@@ -173,29 +174,11 @@ def submit_for_beta_review!(metadata)
           build_number: self.build_version,
           platform: self.platform,
 
-          # Required Metadata:
-          changelog: "No changelog provided",
-          description: "No app description provided",
-          feedback_email: "contact@company.com",
-          marketing_url: "http://marketing.com",
-          first_name: "Felix",
-          last_name: "Krause",
-          review_email: "contact@company.com",
-          phone_number: "0123456789",
-          significant_change: false,
-
-          # Optional Metadata:
-          privacy_policy_url: nil,
-          review_user_name: nil,
-          review_password: nil,
-          encryption: false
```

Comment:
Someone with more knowledge might be able to help here - in my testing this data was populated automatically with either data from the store listing, or data from previous test builds.

Either way it seems like a bad idea to potentially populate this with the placeholder data I've removed? Wouldn't it be better to error?